arm-resources/resources/2016-07-01/swagger/resources.json

```diff
@@ -1684,7 +1689,8 @@
           },
           "description": "The api versions."
         }
-      }
+      },
+      "description": "The type of alias paths. "
```

Comment:
It's alias paths (plural). Basically an alias has paths, and this is the type for each of the path. I can update to "the type of the paths for the alias" but not sure if it looks better.